packages/paste-core/components/topbar/__tests__/TopbarProductSwitcher.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {render, screen, act} from '@testing-library/react';
+
+import {ProductSwitcherMenu} from '../stories/TopbarProductSwitcher/ProductSwitcher.stories';
+import {
+  DefaultElementName,
+  CustomElementName,
+} from '../stories/TopbarProductSwitcher/ProductSwitcher.customization.stories';
+
+describe('TopbarProductSwitcher', () => {
+  describe('element naming', () => {
+    it('should set all default element names', async () => {
+      await act(async () => {
+        render(<ProductSwitcherMenu />);
+      });
+      expect(screen.getByRole('button', {name: 'Switch products'}).dataset.pasteElement).toEqual(
```

Comment:
question: doesn't verifying custom styles on these elements also double as verifying the `element` string is correct? Just wondering if these may be redundant

Reply:
It does, but what the other tests ensure more explicitly is that we expect the name of the element not to change as that a public API